dev/tests/integration/framework/Magento/TestFramework/Mail/TransportInterfaceMock.php

```diff
@@ -35,6 +40,7 @@ public function __construct($message = null)
      */
     public function sendMessage()
```

Comment:
We can have a return type here as `void` maybe
```suggestion
    public function sendMessage() : void
```